Tools/Scripts/webkitpy/port/darwin.py

```diff
@@ -53,10 +53,32 @@ def __init__(self, host, port_name, **kwargs):
             # with MallocStackLogging enabled.
             self.set_option_default("batch_size", 1000)
 
+    def _load_api_test_suite_allowlist(self):
+        """Load the allowlist of test suites that should not go to the system shard."""
+        allowlist_path = os.path.join(os.path.dirname(__file__), '..', 'api_tests', 'allowlist.txt')
+        if not self._filesystem.exists(allowlist_path):
+            return set()
+
+        allowlist = set()
```

Comment:
Using a set is almost always correct, but I think in this case, you want to either use a traditional list or sort the set before returning it as a list. Python does not predictably order sets, and as you have learned, API tests are sometimes inappropriately dependent on ordering.